packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -14,6 +14,12 @@ import 'feedback.dart';
 import 'theme.dart';
 import 'tooltip_theme.dart';
 
+/// The default value used when a value is not provided anywhere.
+///
+/// Read the documentation on the following properties to determine all sources:
```

Comment:
We have a pattern we use for this. There are other examples in docs if you search for `See also:`